src/components/InstrumentControls/Telescope.vue

```diff
@@ -84,14 +84,13 @@
       </p>
     </b-field>
 
-    <b-dropdown aria-role="list" style="width: 100%;" size="is-small">
+    <b-dropdown aria-role="list" style="width: 100%;" size="is-small" scrollable>
       <button class="button is-small" slot="trigger" style="width: 100%;">
           <span>Slew to...</span>
           <b-icon icon="menu-down"></b-icon>
       </button>
       <template
         v-for="(command, idx) in [
-            mount_slew_chart_command,
```

Comment:
Good question. Instead of using this form, I just imported the telescope controls form used in the site observe page. So the functionality is all still there.